app/accept_invite/rest.py

```diff
@@ -38,6 +38,9 @@ def validate_invitation_token(invitation_type, token):
                   ['Your invitation to GOV.UK Notify has expired. '
                    'Please ask the person that invited you to send you another one']}
         raise InvalidRequest(errors, status_code=400)
+    except BadData:
+        errors = {'invitation': 'There’s something wrong with the link you’ve used.'}
```

Comment:
Why are people hitting this error? Is it because they haven’t copy/pasted the whole URL?

Reply:
yeah

Reply:
The error message should help them fix that then…